### PR TITLE
S3E5

### PR DESCRIPTION
**Pull request by ArronHZG**
2019-09-08-seminar

### CODE REVIEW

File: docs/_posts/2019-09-08-seminar-meeting-minutes-3-5.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+author: gaojiuy
+date: 2019-09-08 19:00 +0800
+comments: true
+mathjax: true
+categories:
+  - events
+tags:
+  - 
+header:
+  teaser: /assets/2019/seminar-3-4/joint-photo1.jpg
```
Comment (CyFeng16) [inline docs/_posts/2019-09-08-seminar-meeting-minutes-3-5.md]: The teaser photo here is wrong.

Comment (ArronHZG) [reply docs/_posts/2019-09-08-seminar-meeting-minutes-3-5.md]: ok ,i got it.

**Comment by VDeamoV** [comment]
Plz close this extra pr

**Comment by newip** [comment]
@ArronHZG Seems the picture already updated with right size. Would you please close the PR if it is duplicated. Tks.

**Pull request closed without merging**